front/src/modules/timesStops/TimesStopsInput.tsx

```diff
@@ -119,11 +121,12 @@ const TimesStopsInput = ({
           (row, index) =>
             !isEqual(normalizeNullablesInRow(row), normalizeNullablesInRow(rows[index]))
         )
-        .map(({ shortSlipDistance, onStopSignal, arrival, departure, ...row }) => ({
+        .map(({ shortSlipDistance, onStopSignal, arrival, departure, stopFor, ...row }) => ({
           ...row,
           arrival: durationSinceStartTime(startTime, arrival),
           departure: durationSinceStartTime(startTime, departure),
           receptionSignal: onStopSignalToReceptionSignal(onStopSignal, shortSlipDistance),
+          stopFor: stopFor ? new Duration(sec2ms(Number(stopFor))) : null,
```

Comment:
We could have a static `Duration.fromSeconds`
this seems like an operation we would need to do often

Reply:
Fine by me !